Move recaptcha config to contact fragment frontmatter

mpourismaiel opened the pull request:
<!--
Thanks for sending a pull request!  
If this is your first time, read our [contributing guidelines](/CONTRIBUTING.md).
-->
**What this PR does / why we need it**:
Move recaptcha config from main config file to fragment config

Also made recaptcha optional with one set-back; Submit button is always enabled.

**Which issue this PR fixes** 
fixes #88

stp-ip commented on exampleSite/content/index/contact.md:
```diff
@@ -17,6 +17,10 @@ email = "mail@example.com"
 #netlify = false
 
+# Optional google captcha
+#[params.contact.captcha]
```
Could probably be simplified to just:
```
#[captcha]
#  sitekey = ""
```
No need to have the more complex format.

stp-ip replied:
Or [recaptcha] to make it clear, that it's a setting for google's recaptcha.

stp-ip commented on static/main.css:
```diff
@@ -9691,9 +9691,11 @@ div.form-error ul li {
 
 #particles-js {
```
Separate change so might usually be better as a separate PR.

mpourismaiel commented:
@stp-ip @lauragift21 I pushed again and removed the previous commit since it was not working properly. I removed the wrong diff in main.css file and fixed a bug in contact.md variable names.

stp-ip commented:
/LGTM from my side.

lauragift21 commented:
Looks good to me too. :+1:

lauragift21 commented on static-main/js/form.js:
```diff
@@ -13,13 +13,13 @@ if ($.validate) {
     onError: function($form) {
       $('form[id=' + $form.attr('id') +'] .generic-error').removeClass('d-none');
-  },
+    },
```
whitespace included here.

mpourismaiel replied:
I don't understand.

mpourismaiel replied:
No I added the whitespace myself. Previously the indent level did not match on this line, every other line started at the 3rd level and this line was not indented.

mpourismaiel commented:
There's a debugger in the code. I don't have access to a laptop write now. I'll fix it in 3-4 hours.

stp-ip commented:
/LGTM to also fix the demo
